cli/chain.go

```diff
@@ -1504,3 +1505,38 @@ var ChainPruneCmd = &cli.Command{
 		return api.ChainPrune(ctx, opts)
 	},
 }
+
+var ChainGetTipSetByHeightCmd = &cli.Command{
+	Name:    "gettipset",
+	Aliases: []string{"love"},
+	Usage:   "View a segment of the chain",
+	Flags: []cli.Flag{
+		&cli.Uint64Flag{Name: "height", DefaultText: "current head"},
```

Comment:
I would suggest making this a mandatory parameter. We already have `lotus chain head` to print the head.